01-js-intro-and-jquery/spotify.js

```diff
@@ -0,0 +1,31 @@
+$(document).ready(function(){
+	
+	$("input").focus();
+
+	var searchList = function(query){
+		$.ajax({
+			type: "GET",
+			url: "https://api.spotify.com/v1/search",
+			data: { q: query, type: 'track' },
+			success: function(response){
+				$("ul").empty();
+				console.log(response);
+				if(response.tracks.items.length > 0){
```

Comment:
This line is the same as 

```
if (response.tracks.items.length) { ... }
```

If the length is 0 will be falsy so this want be executed